app/jsontableschema/constraints.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
```

Comment:
@zhenyab why is this committed? it looks like transpiled files from babel, right?